.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,7 +2,7 @@
 name: Feature request
 about: Suggest an idea for this project
 title: '<short description for the feature>'
-labels: 'enhancement'
+labels: 'type:enhancement'
```

Comment:
I'm conflicted between the enhancement and the proposal actually.

Reply:
Good point.  Every open issue with label `type:enhancement` currently _also_ has `type:proposal` (enhancement implies proposal).  The reverse is not true (some open proposals are enhancements, but not all).

Reply:
As such I think we should use proposal instead and we will change to enhancement if we agree about it.

Reply:
Agreed - thank you!